4.DZ/date.hpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+namespace DZ4 {
+
+struct Date {
+	int day;
+	int month;
+	int year;
+};
+
+void run() {
```

Comment:
Please add description what this function does with following format
/**
/* @description: your description
*/

Reply:
CMakeFile.txt was added